lib/config/contributors.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const YAML = require('yaml').default;
+const fs = require('../deps/fsp.js');
+const path = require('path');
+
+const CONTRIBUTORS_CONFIG_LOCATION = path.resolve(__dirname, '..', '..', 'contributors', '_contributors.yaml');
+const CONTRIBUTORS_PICTURES_LOCATION = path.resolve(__dirname, '..', '..', 'content', 'images', 'contributors');
+
+let authorsConfig;
+
+async function loadContributorFromConfiguration(author) {
+    if (!authorsConfig) {
```

Comment:
Do you need this check? Will this always be undefined when this function is called?

Reply:
It's to lazily compute the map, as `fs.readFile` is async. The assignment is on line 31.

Reply:
ah got it. I didn't realize this function was being called over and over and preserving the state.